packages/components/divider/src/divider/intents.tsx

```diff
@@ -0,0 +1,26 @@
+import { tw } from '@spark-ui/internal-utils'
```

Comment:
Shouldn't the directory be named `variants` instead of `divider`?

Reply:
you are right